src/transformStateWithClones.js

```diff
@@ -7,7 +7,25 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = Object.assign({}, state);
```

Comment:
The variable name `currentState` is appropriate as per the checklist item 5. Good job on that!